src/main/java/sage/utility/Command.java

```diff
@@ -0,0 +1,56 @@
+package sage.utility;
+
+public class Command {
+    private String[] rawOutput;
```

Comment:
Since it is an array try to use the plural form
```suggestion
    private String[] rawOutputs;
```